src/functional.ts

```diff
@@ -136,6 +179,103 @@ export function functionalInterfaceFactory({
     }
   }
 
+  function convertToBlstPublicKeyArg(publicKey: PublicKeyArg): blst.PublicKeyArg {
```

Comment:
prefer helper functions stay in the last

Reply:
Updated.